content/700-optimize/200-getting-started.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Getting Started'
+metaTitle: 'Getting started with Prisma Optimize'
+metaDescription: 'Learn how to quickly set up and start using Prisma Optimize.'
+tocDepth: 3
+toc: true
+---
+
+## Prerequisites
+
+Before you begin with Prisma Optimize, ensure you have the following:
+
+- A [Prisma Data Platform account](https://console.prisma.io).
+- A project using [Prisma Client](/orm/prisma-client) version `5.0.0` or higher (we recommend using the latest version).
+- A PostgreSQL, MySQL/MariaDB, PlanetScale, CockroachDB, MS SQL Server, or MongoDB database.
+
+## 1. Launch Optimize
+
+1. Log in to your [Prisma Data Platform account](https://console.prisma.io/optimize).
+2. Access and launch the Optimize dashboard by following the instructions [here](/platform/about#accessing-the-optimize-dashboard).
+
+## 2. Add Optimize to your application
+
+### 2.1. Update Your `schema.prisma` file
+
+In the `generator` block of your Prisma schema, add the `tracing` preview feature:
+
+```prisma
+generator client {
+  provider = "prisma-client-js"
+  previewFeatures = ["tracing"]
+}
+```
+
+Then, generate the Prisma Client:
+
+```bash
+npx prisma generate
+```
+
+### 2.2. Install the Optimize Prisma Client extension
+
+Install the latest versions of Prisma Client and the Optimize extension:
+
+```bash
+npm install @prisma/client@latest @prisma/extension-optimize
+```
+
+### 2.3. Add the Optimize API Key to your `.env` file
+
+Generate an Optimize API key by following the instructions [here](/platform/about#generating-an-optimize-api-key) and add it to your `.env` file:
+
+```bash
+OPTIMIZE_API_KEY="YOUR_OPTIMIZE_API_KEY"
+```
+
+### 2.4. Extend your Prisma Client instance with the Optimize extension
+
+Extend your existing Prisma Client instance with the Optimize extension:
+
+```ts
+import { PrismaClient } from "@prisma/client";
+import { withOptimize } from "@prisma/extension-optimize";
+
+const prisma = new PrismaClient().$extends(
+  withOptimize({ token: process.env.OPTIMIZE_API_KEY }),
+);
+```
+
+### 2.5. Use Prisma Optimize to generate insights
+
+Follow these steps to start generating query insights with Prisma Optimize:
+
+1. In the Optimize dashboard, click the **Start recording** button, then run your app.
+2. After your app runs and insights are generated for the desired queries, click the **Stop recording** button.
```

Comment:
this feels like i can only run it for 1 thing. Somehow change this to be like "do anything in your app that runs queries on the prisma client and see the dashboard fill, once you feel you have a proper dataset stop the recoding" or something like that. Thats way too many words but ukwim